modules/context/repo.go

```diff
@@ -103,6 +103,14 @@ func (r *Repository) CanUseTimetracker(issue *models.Issue, user *models.User) b
 		r.IsWriter() || issue.IsPoster(user.ID) || issue.AssigneeID == user.ID)
 }
 
+// CanUseDependencies returns whether or not a user can create dependencies.
+func (r *Repository) CanUseDependencies(issue *models.Issue, user *models.User) bool {
```

Comment:
Why not named it `CanCreateIssueDepend`?

Reply:
Makes sense, I'll change it.

Reply:
Done.